src/pip/_internal/wheel.py

```diff
@@ -204,6 +204,26 @@ def message_about_scripts_not_on_PATH(scripts):
     return "\n".join(msg_lines)
 
 
+def sorted_outrows(outrows):
+    """
+    Return the given rows of a RECORD file in sorted order.
+
+    Each row is a 3-tuple (path, hash, size) and corresponds to a record of
+    a RECORD file (see PEP 376 and PEP 427 for details).  For the rows
+    passed to this function, the size can be an integer as an int or string,
+    or the empty string.
+    """
+    # Normally, there should only be one row per path, so the second and
+    # third elements of each row don't normally come into play when sorting.
```

Comment:
Two “normally” in this sentence.

Reply:
Do you have a suggestion of how it should be rephrased? The meaning doesn't seem correct to me if either one is removed.

Reply:
Maybe “Normally, there should …, and the second and third in this case …”?

Reply:
K